app/components/windows/SceneTransitions.vue.ts

```diff
@@ -144,6 +144,8 @@ export default class SceneTransitions extends Vue {
   }
 
   getSceneName(id: string) {
+    if (id === 'ALL') return $t('All');
```

Comment:
`getSceneName(id: string |  'ALL')`
I'm also sugesting use `*` instead `ALL`

Reply:
hmm why do you suggest that?

Reply:
also not sure why we need to type differently here since id will never not be a string

Reply:
Using `*` is just a common solution to specify a wildcard in many libs.  It also allows to add more flexible filters like a partial match in the future: `my-scene-*` <-- match all scenes names starting with `my-scene-`.

`getSomething(id: string | 'ALL') ` <-- will give you better intelliSence for understanding that method uses some reserved keywords
![image](https://user-images.githubusercontent.com/3768346/55178712-252ee600-5143-11e9-8284-49377f1fe21c.png)

Reply:
right now we aren't actually using any matching for this though, nor do i think this is a use-case where we would have partial matching. the user isn't inputting IDs for us to match, we're programmatically generating them without any distinguishing features. if we do allow user-generated IDs in the future i still don't think we'll be able to usefully implement partial matching since we'll have no way to enforce what users submit as an ID while still allowing for useful flexibility